codegen/Cargo.toml

```diff
@@ -25,11 +26,12 @@ quote = { workspace = true }
 syn = { workspace = true }
 scale-info = { workspace = true }
 subxt-metadata = { workspace = true }
-jsonrpsee = { workspace = true, features = ["async-client", "client-ws-transport-tls", "http-client"], optional = true }
+jsonrpsee = { workspace = true, features = ["ws-client", "http-client"], optional = true }
```

Comment:
I guess this means that, without importing `jsonrpsee` myself to enable TLS features, that we won't be able to use fetch-metadata with httpS urls?

Reply:
yes without jsonrpsee it will only work to fetch metadata from a file or runtime path...

Reply:
ooooh, tls enabled by default by the ws-client and http-client :P